app/src/main/res/values-v21/styles.xml

```diff
@@ -31,4 +31,4 @@
     <item name="android:progressBackgroundTint">@color/greyed_out_selected</item>
     <item name="android:statusBarColor">@android:color/transparent</item>
   </style>
-</resources>
```

Comment:
Please don't commit files you haven't touched. Please remove this from your commit and repush.

Reply:
@mhutti1  I did not change this. Although I noticed this before and tried out but I got this error 
_Commit failed with error
									0 files committed, 1 file failed to commit: Disable swipeLayout after long press
									warning: LF will be replaced by CRLF in app/src/main/res/values-v21/styles.xml.
									The file will have its original line endings in your working directory_

Actually, it happened maybe because your and mine are different platform ( Unix/ Windows) but still I don't think it makes much difference.
You can refer for the same here
https://stackoverflow.com/questions/5834014/lf-will-be-replaced-by-crlf-in-git-what-is-that-and-is-it-important